packages/visualizations/src/components/Map/Svg/Choropleth.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+    import { geoEqualEarth } from 'd3-geo';
+    import { colorShapes, LIGHT_GREY, DEFAULT_COLORSSCALE, EMPTY_GJ } from '../utils';
+    import Map from './Map.svelte';
+
+    import type { SvgChoroplethOptions, ChoroplethDataValue } from '../types';
+
+    export let data: { value: ChoroplethDataValue[] };
+    export let options: SvgChoroplethOptions;
+    $: ({ geoJson, colorsScale = DEFAULT_COLORSSCALE, emptyValueColor = LIGHT_GREY } = options);
+
+    $: coloredShapes =
+        geoJson?.geoJson && geoJson.type === 'geojson'
+            ? colorShapes(geoJson.geoJson, data.value, colorsScale, emptyValueColor)
```

Comment:
The `mapKeyToColor` is actually all we need here, since we re-extract the color from the geoJson later on (with something a bit cumbersome tbh). We should wait for #105 to be merged and then move to `mapKeyToColor`.

Reply:
👍